test/topology_random_failures/cluster_events.py

```diff
@@ -0,0 +1,624 @@
+                   random_tables: RandomTables,
+                   error_injection: str) -> AsyncIterator[None]:
+    udt_name = "test_random_failures_udt_to_drop"
+
+    LOGGER.info("Add an UDT to drop")
+    await random_tables.add_udt(name=udt_name, cmd="(a text, b int)")
+
+    yield
+
+    LOGGER.info("Drop an UDT from the schema")
+    await random_tables.drop_udt(name=udt_name)
+
+    yield
+
+    LOGGER.info("Check if the UDT was dropped by re-creating it")
+    await random_tables.add_udt(name=udt_name, cmd="(a text, b int)")
+
+
+async def insert_records(manager: ManagerClient,
+                         random_tables: RandomTables,
+                         error_injection: str) -> AsyncIterator[None]:
+    yield
+
+    LOGGER.info("Add records to the table")
+    await random_tables[0].insert_seq()
+
+    yield
+
+
+async def update_record(manager: ManagerClient,
+                        random_tables: RandomTables,
+                        error_injection: str) -> AsyncIterator[None]:
+    LOGGER.info("Add a record to the table")
+    table = random_tables[0]
+    await table.insert_seq()
+
+    yield
+
+    pk_columns = table.columns[:table.pks]
+    set_column = table.columns[-1]
+    next_seq = table.next_seq()
+    set_value = set_column.val(seed=next_seq)
+
+    LOGGER.info("Update the record in the table")
+    await manager.cql.run_async(
+        f"UPDATE {table.full_name} SET {set_column} = %s WHERE {' AND '.join(f'{c.name} = %s' for c in pk_columns)}",
+        [set_value] + [c.val(next_seq-1) for c in pk_columns],
+    )
+
+    yield
+
+
+@deselect_for(
+    # TODO: remove this skip when #18068 will be resolved.
+    reason="LWT is not yet supported with tablets. See issue #18068"
+)
+async def execute_lwt_transaction(manager: ManagerClient,
+                                  random_tables: RandomTables,
+                                  error_injection: str) -> AsyncIterator[None]:
+    LOGGER.info("Add a record to the table")
+    table = random_tables[0]
+    await table.insert_seq()
+
+    yield
+
+    pk_columns = table.columns[:table.pks]
+    set_column = table.columns[-1]
+    next_seq = table.next_seq()
+    set_value = set_column.val(seed=next_seq)
+
+    LOGGER.info("Execute a lightweight transaction")
+    await manager.cql.run_async(
+        f"UPDATE {table.full_name}"
+        f" SET {set_column} = %s"
+        f" WHERE {' AND '.join(f'{c.name} = %s' for c in pk_columns)}"
+        f" IF EXISTS",
+        [set_value] + [c.val(next_seq-1) for c in pk_columns],
+    )
+
+    yield
+
+
+@deselect_for(
+    error_injections=[
+        "stop_after_starting_auth_service",
+        "stop_after_setting_mode_to_normal",
+        "stop_before_becoming_raft_voter",
+        "stop_after_updating_cdc_generation",
+        "stop_before_streaming",
+        "stop_after_streaming",
+    ],
+    reason="See issue #19151 (decommission process stuck while boostrapping node is paused)",
```

Comment:
#19151 describes expected behavior. So the event or test should probably get adjusted somehow to account for that.

Reply:
We decided to leave it outside of this PR.